package.json

```diff
@@ -24,6 +24,7 @@
   },
   "homepage": "https://github.com/ph0bos/express-microservice-starter",
   "dependencies": {
+    "address": "^1.0.3",
```

Comment:
The other packages in here specify a concrete version, worth just using "1.0.3"?